plugins/out_loki/loki.c

```diff
@@ -320,7 +338,17 @@ static flb_sds_t pack_labels(struct flb_loki *ctx, msgpack_packer *mp_pck,
                 /* append the key/value pair */
                 flb_mp_map_header_append(&mh);
                 msgpack_pack_str(mp_pck, k.via.str.size);
+
+#if FLB_LOKI_ESCAPE_KUBERNETES_AUTO_KEYS == 1
+                char *dkey = strndup(k.via.str.ptr, k.via.str.size);
+                escape_utf8_char_sequence(dkey, k.via.str.size);
```

Comment:
we can manage this for normal strings, since the chars we are checking are normal ASCII bytes